deployment/CP04OSSM_Deploy.csv

```diff
@@ -404,65 +404,65 @@ AR52,,AR61,,CP04OSSM-SBD12-00-DCLENG000,13,1,2021-03-30T15:02:00,2021-11-04T12:1
+AR66,,,,CP04OSSM-MFD37-03-CTDBPE000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-CTDBPE-50067,39.937517,-70.88685,,453,454,,,
+AR66,,,,CP04OSSM-MFD37-04-DOSTAD000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-DOSTAD-00392,39.937517,-70.88685,,453,454,,,
+AR66,,,,CP04OSSM-MFD37-07-ZPLSCC000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-ZPLSCC-55100,39.937517,-70.88685,,453,454,,,
+AR66,,,,CP04OSSM-RIC21-00-CPMENG000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,3703-00051-00001-01077,39.937517,-70.88685,,7,454,,,
+AR66,,,,CP04OSSM-RID26-00-DCLENG000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,3718-00001-00001-00058,39.937517,-70.88685,,7,454,,,
+AR66,,,,CP04OSSM-RID26-04-VELPTA000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-VELPTA-12230,39.937517,-70.88685,,7,454,,,
+AR66,,,,CP04OSSM-RID26-06-PHSEND000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-PHSEND-P0095,39.937517,-70.88685,,7,454,,,
+AR66,,,,CP04OSSM-RID26-07-NUTNRB000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-NUTNRB-01094,39.937517,-70.88685,,7,454,,,
+AR66,,,,CP04OSSM-RID26-08-SPKIRB000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-SPKIRB-00291,39.937517,-70.88685,,7,454,,,
+AR66,,,,CP04OSSM-RID27-00-DCLENG000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,3718-00001-00001-00061,39.937517,-70.88685,,7,454,,,
+AR66,,,,CP04OSSM-RID27-01-OPTAAD000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-OPTAAD-00123,39.937517,-70.88685,,7,454,,,
+AR66,,,,CP04OSSM-RID27-02-FLORTD000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-FLORTD-01216,39.937517,-70.88685,,7,454,,,
+AR66,,,,CP04OSSM-RID27-03-CTDBPC000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-CTDBPC-06841,39.937517,-70.88685,,7,454,,,
+AR66,,,,CP04OSSM-RID27-04-DOSTAD000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-DOSTAD-00387,39.937517,-70.88685,,7,454,,,
+AR66,,,,CP04OSSM-SBC11-00-CPMENG000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,3703-00051-00001-01072,39.937517,-70.88685,,0,454,CPM secondary 3703-00051-00001-01073 ,,
+AR66,,,,CP04OSSM-SBD11-00-DCLENG000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,3718-00001-00001-00014,39.937517,-70.88685,,0,454,,,
+AR66,,,,CP04OSSM-SBD11-01-MOPAK0000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-MOPAK0-42298,39.937517,-70.88685,,0,454,,,
+AR66,,,,CP04OSSM-SBD11-02-HYDGN0000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-HYDGN0-49886,39.937517,-70.88685,,0,454,,,
+AR66,,,,CP04OSSM-SBD11-06-METBKA000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-METLGR-00037,39.937517,-70.88685,,0,454,,,
+AR66,,,,CP04OSSM-SBD12-00-DCLENG000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,3718-00001-00001-00051,39.937517,-70.88685,,0,454,,,
+AR66,,,,CP04OSSM-SBD12-03-HYDGN0000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-HYDGN0-49857,39.937517,-70.88685,,0,454,,,
+AR66,,,,CP04OSSM-SBD12-04-PCO2AA000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-PCO2AA-34233,39.937517,-70.88685,,0,454,,,
```

Comment:
deployment time/location, recovery time, and deploy/water depth updates look good. configuration previously reviewed and approved.

Reply:
All updates look good. Approved.